cloud_function/migrate_database.py

```
@@ -0,0 +1,586 @@
+        log(f"[OK] Discord notification sent: {title}")
+        
+    except Exception as e:
+        log(f"[WARNING] Failed to send Discord notification: {e}")
+
+def get_connection(db_url, dict_cursor=False):
+    """Create a database connection"""
+    try:
+        conn = psycopg2.connect(db_url)
+        if dict_cursor:
+            conn.cursor_factory = RealDictCursor
+        return conn
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)